package.json

```diff
@@ -37,6 +37,7 @@
     "@radix-ui/react-label": "^2.1.0",
     "@radix-ui/react-slot": "^1.1.0",
     "@react-email/components": "^0.0.25",
+    "@sentry/react": "^8.32.0",
```

Comment:
**question:** Not sure if `react` package is correct, because right now Sentry is used in API, which is on bun.

https://docs.sentry.io/platforms/javascript/guides/bun/